tooling/vscode-fcp/python/lint.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python3
```

Comment:
Learning: what does this file do? What sort of diagnostics do we get? Syntax errors?